contents/docs/runbook/async-migrations/index.md

```diff
@@ -113,17 +113,24 @@ The table below lists out recommended PostHog app and chart versions to use for
 | 0004            | 1.36.1          | 26.0.0        | Run the async migration right after upgrading as there could be problems with ingestion otherwise | 
 
 
-#### Upgrading hobby deployment to a specific version
+#### Upgrading helm chart to a specific version
 
-Before following the normal upgrading procedure update the `.env` file to have `POSTHOG_APP_TAG` match `release-<desired version>`. For example run
+To upgrade to a specific PostHog app version specify the desired version in your `values.yaml`
 ```
-echo "POSTHOG_APP_TAG=release-1.33.0" >>.env
+image:
+  tag: release-1.36.1
 ```
 
-#### Upgrading helm chart to a specific version
-
 To upgrade to a specific chart version you can use `--version <desired version>` flag, e.g.
 ```
 helm upgrade -f values.yaml --timeout 30m --namespace posthog posthog posthog/posthog --atomic --wait --wait-for-jobs --debug --version 16.1.0
 ```
 Make sure you have followed the [upgrade instructions](https://posthog.com/docs/runbook/upgrading-posthog) for your platform (specifically major upgrade notes as needed).
+
+
+#### Upgrading hobby deployment to a specific version
+
+Before following the normal upgrading procedure update the `.env` file to have `POSTHOG_APP_TAG` match `release-<desired version>`. For example run
+```
+echo "POSTHOG_APP_TAG=release-1.33.0" >>.env
+```
```

Comment:
Not actually sure if this is true anymore, probably best to link to [here](https://posthog.com/docs/self-host/deploy/hobby#upgrading) in the hobby deployment docs. The upgrade script I believe asks what version you would like to update to

Reply:
https://github.com/PostHog/posthog/blob/master/bin/upgrade-hobby#L52 that flag is needed, but I agree that it would be better to link to that page instead